src/Tasks/DownloadFile.cs

```diff
@@ -181,7 +185,11 @@ private async Task DownloadAsync(Uri uri, CancellationToken cancellationToken)
                         {
                             Log.LogMessageFromResources(MessageImportance.High, "DownloadFile.Downloading", SourceUrl, destinationFile.FullName, response.Content.Headers.ContentLength);
 
+#if NET5_0_OR_GREATER
+                            using (Stream responseStream = await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false))
```

Comment:
core: warning when we don't pass the cancellationtoken

framework: ReadStreamAsync doesn't have an overload that takes the token